docs/getting-started-apm-server.asciidoc

```diff
@@ -232,10 +232,36 @@ Any changes are automatically appended to the `apm-server.yml` configuration fil
 Full details are available in the {cloud}/ec-manage-apm-settings.html[APM user settings] documentation.
 
 [float]
-[[index-template-config]]
 ==== Self installation
 
-It is recommend that you run the `setup` command before starting {beatname_uc}.
+[float]
+===== APM integration for Elastic Agent
```

Comment:
Maybe also link to https://www.elastic.co/guide/en/apm/server/current/apm-integration.html?